packages/ts-config/main.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "main-tsconfig",
+  "compilerOptions": {
+      "target": "ESNext",
+      "moduleResolution": "node",
+      "module": "esnext",
+      "outDir": "dist",
+      "incremental": true,
+      "composite": true,
+      "declaration": true,
+      "resolveJsonModule": true,
+      "removeComments": true,
+      "emitDecoratorMetadata": true,
+      "experimentalDecorators": true,
+      "esModuleInterop": true,
+      "allowSyntheticDefaultImports": true,
+      "skipLibCheck": true,
+      "noImplicitAny": true,
+      "strictNullChecks": true,
+      "strictPropertyInitialization": false,
```

nit: you have set `strictPropertyInitialization` to `false`. This can be useful for initial development, but to maintain security and help identify potential bugs, it is important to set it to `true`. This forces you to explicitly initialize all class properties.

```suggestion
      "strictPropertyInitialization": true,
```

---

I believe it's unnecessery ❤️